util/retrieval.go

```diff
@@ -15,3 +19,25 @@ type RetrievalProgress struct {
 	Wait   chan struct{}
 	EndErr error
 }
+
+type HeartbeatAutoretrieveResponse struct {
+	Handle         string    `json:"handle"`
+	LastConnection time.Time `json:"lastConnection"`
+}
+
+type AutoretrieveListResponse struct {
```

Comment:
the `AutoretrieveListResponse` and `InitAutoretrieveResponse` looks to have the same structure. Can we just merge them into `AutoRetrieveResponse` or is there a particular reason for it? e.i is `AutoretrieveListResponse` meant to be a [] list?

Reply:
I was following the same structure (read: copy pasting) of the shuttle register and listing api. In the future we might want these two responses to be different.